internal/btf/btf.go

```diff
@@ -152,7 +152,7 @@ func loadSpecFromVmlinux(rd io.ReaderAt) (*Spec, error) {
 
 	btfSection, _, _, err := findBtfSections(file)
 	if err != nil {
-		return nil, fmt.Errorf(".BTF ELF section: %s", err)
+		return nil, fmt.Errorf(".BTF ELF section: %w", err)
```

Comment:
```diff
diff --git a/internal/btf/core.go b/internal/btf/core.go
index 7c888f6..a31abf2 100644
--- a/internal/btf/core.go
+++ b/internal/btf/core.go
@@ -473,7 +473,7 @@ func parseCoreAccessor(accessor string) (coreAccessor, error) {
                // 31 bits to avoid overflowing int on 32 bit platforms.
                index, err := strconv.ParseUint(part, 10, 31)
                if err != nil {
-                       return nil, fmt.Errorf("accessor index %q: %s", part, err)
+                       return nil, fmt.Errorf("accessor index %q: %w", part, err)
                }
 
                result = append(result, int(index))
@@ -559,7 +559,7 @@ func coreFindField(local Type, localAcc coreAccessor, target Type) (_, _ coreFie
                        if localMember.Name == "" {
                                _, ok := localMember.Type.(composite)
                                if !ok {
-                                       return coreField{}, coreField{}, fmt.Errorf("unnamed field with type %s: %s", localMember.Type, ErrNotSupported)
+                                       return coreField{}, coreField{}, fmt.Errorf("unnamed field with type %s: %w", localMember.Type, ErrNotSupported)
                                }
 
                                // This is an anonymous struct or union, ignore it.
diff --git a/internal/btf/ext_info.go b/internal/btf/ext_info.go
index beba1bc..893c8db 100644
--- a/internal/btf/ext_info.go
+++ b/internal/btf/ext_info.go
@@ -269,7 +269,7 @@ func parseExtInfoRelos(r io.Reader, bo binary.ByteOrder, strings stringTable) (m
 
                        accessor, err := parseCoreAccessor(accessorStr)
                        if err != nil {
-                               return nil, fmt.Errorf("accessor %q: %s", accessorStr, err)
+                               return nil, fmt.Errorf("accessor %q: %w", accessorStr, err)
                        }
 
                        relos = append(relos, coreRelo{
diff --git a/internal/btf/types.go b/internal/btf/types.go
index 9b7eca2..812d2a9 100644
--- a/internal/btf/types.go
+++ b/internal/btf/types.go
@@ -766,7 +766,7 @@ func inflateRawTypes(rawTypes []rawType, rawStrings stringTable) (types []Type,
                        for i, btfVal := range rawvals {
                                name, err := rawStrings.LookupName(btfVal.NameOff)
                                if err != nil {
-                                       return nil, nil, fmt.Errorf("get name for enum value %d: %s", i, err)
+                                       return nil, nil, fmt.Errorf("get name for enum value %d: %w", i, err)
                                }
                                vals = append(vals, EnumValue{
                                        Name:  name,
@@ -813,7 +813,7 @@ func inflateRawTypes(rawTypes []rawType, rawStrings stringTable) (types []Type,
                        for i, param := range rawparams {
                                name, err := rawStrings.LookupName(param.NameOff)
                                if err != nil {
-                                       return nil, nil, fmt.Errorf("get name for func proto parameter %d: %s", i, err)
+                                       return nil, nil, fmt.Errorf("get name for func proto parameter %d: %w", i, err)
                                }
                                params = append(params, FuncParam{
                                        Name: name,
diff --git a/internal/elf.go b/internal/elf.go
index 54a4313..88326bb 100644
--- a/internal/elf.go
+++ b/internal/elf.go
@@ -24,7 +24,7 @@ func NewSafeELFFile(r io.ReaderAt) (safe *SafeELFFile, err error) {
                }
 
                safe = nil
-               err = fmt.Errorf("reading ELF file panicked: %s", r)
+               err = fmt.Errorf("reading ELF file panicked: %w", r)
        }()
 
        file, err := elf.NewFile(r)
@@ -44,7 +44,7 @@ func (se *SafeELFFile) Symbols() (syms []elf.Symbol, err error) {
                }
 
                syms = nil
-               err = fmt.Errorf("reading ELF symbols panicked: %s", r)
+               err = fmt.Errorf("reading ELF symbols panicked: %w", r)
        }()
 
        syms, err = se.File.Symbols()
@@ -60,7 +60,7 @@ func (se *SafeELFFile) DynamicSymbols() (syms []elf.Symbol, err error) {
                }
 
                syms = nil
-               err = fmt.Errorf("reading ELF dynamic symbols panicked: %s", r)
+               err = fmt.Errorf("reading ELF dynamic symbols panicked: %w", r)
        }()
 
        syms, err = se.File.DynamicSymbols()
```